### DISCUSSION TITLE
Can't open multiple collections on the same database at the same time

### DISCUSSION THREAD

**Discussion by yvonneekanim** [specify/specify7#5520] [Feature Requests]
Currently a user can only log into one collection in the same database at a time. This poses an issue when trying to compare data from two collections especially those with collection relationships.

**Comment by maxpatiiuk** [comment]
The fix would be related to https://github.com/specify/specify7/issues/2902 and https://github.com/specify/specify7/issues/1886

**Comment by grantfitzsimmons** [comment]
Right now this can be accomplished by using two different browsers or using incognito/private browsing modes.

**Comment by maxpatiiuk** [comment]
There are also browser extensions that can help with this:
- https://chrome.google.com/webstore/detail/sessionbox-multi-login-to/megbklhjamjbcafknkgmokldgolkdfig
- https://addons.mozilla.org/en-US/firefox/addon/multi-account-containers/
- there are probably more too

**Comment by grantfitzsimmons** [comment]
I believe we shouldn't handle this in a special way. For those who want to do this, using a second browser, incognito window, or one of the browser extensions mentioned by Max seems sufficient. Maybe we should consider closing this– @maxpatiiuk what are your thoughts?

**Comment by maxpatiiuk** [comment]
Fixing this would require back-end changes and front-end changes

Also, I just realized that this issue might be a duplicate of https://github.com/specify/specify7/issues/2902 (depending on the implementation). And of course there is a "beast" https://github.com/specify/specify7/issues/1886

I would suggest we delay fixing these until decoupling from sp6 when we could sit down and properly rethink the future of scoping (and scoping customization)
